packages/runtime/runtime-utils/src/summarizerNode/summarizerNodeUtils.ts

```diff
@@ -288,3 +288,30 @@ export interface ICreateChildDetails {
     /** Sequence number of latest known change to the node */
     changeSequenceNumber: number;
 }
+
+export interface ISubtreeInfo {
+    /** Tree to use to find children subtrees */
+    childrenTree: ISnapshotTree,
+    /** Additional path part where children are isolated */
+    childrenPathPart: string | undefined,
+}
+
+/**
+ * Checks if the summary contains .channels subtree where the children subtrees
+ * would be located if exists.
+ * @param baseSummary - summary to check
+ */
+export function parseSummaryForSubtrees(baseSummary: ISnapshotTree): ISubtreeInfo {
+    // New versions of snapshots have child nodes isolated in .channels subtree
+    const channelsSubtree = baseSummary.trees[channelsTreeName];
```

Comment:
>channelsTreeName [](start = 46, length = 16)

this seem coupled to our specific container runtime. i feel like this should live in container runtime

Reply:
I agree, but will need to revisit later to extract this functionality.